rotkehlchen/globaldb/schema.py

```diff
@@ -118,20 +118,20 @@
     coingecko TEXT,
     cryptocompare TEXT,
     forked TEXT,
-    FOREIGN KEY(forked) REFERENCES assets(identifier) ON UPDATE CASCADE
+    FOREIGN KEY(forked) REFERENCES assets(identifier) ON UPDATE CASCADE ON DELETE CASCADE,
```

Comment:
This seems wrong. You shouldn't be able to remove entries if you delete the forked entry. Why change the behavior here?